Gemfile

```diff
@@ -235,7 +235,8 @@ gem 'paranoia'
 gem 'petit', github: 'code-dot-org/petit'  # For URL shortening
 
 # JSON model serializer for REST APIs.
-gem 'active_model_serializers', github: 'rails-api/active_model_serializers', ref: '2962f3f64e7c672bfb5a13a8f739b5db073e5473'
+#gem 'active_model_serializers', github: 'rails-api/active_model_serializers', ref: '2962f3f64e7c672bfb5a13a8f739b5db073e5473'
+gem 'active_model_serializers', '~> 0.10.0'
```

Comment:
I'm on board with this approach, but heads up that moving off of the forked serializer is likely to be its own chunk of work. https://codedotorg.atlassian.net/browse/XTEAM-29 contains a very small amount of additional information. @Hamms may have a sense of what blocked this upgrade from being merged to staging in https://github.com/code-dot-org/code-dot-org/pull/26661, and/or how much work likely remained at that time. this work seems worthwhile, but if it was going to take more than a week or two then we might want to reevaluate.

Reply:
Interesting. I'll run unit tests for this as a start to see what fails.

Reply:
I mentioned this in a DM, but repeating here for discoverability:

there ended up being far more failures than I expected, and I recall being surprised at how complicated some of the fixes were shaping up to be, but unfortunately I don't remember any specifics

Reply:
https://github.com/code-dot-org/code-dot-org/pull/36560